src/main/java/com/frcteam3636/frc2024/Dashboard.kt

```diff
@@ -4,10 +4,32 @@ import com.frcteam3636.frc2024.subsystems.drivetrain.Drivetrain
 import com.frcteam3636.frc2024.subsystems.shooter.Shooter
 import edu.wpi.first.wpilibj.RobotController
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard
+import java.net.InetAddress
 import kotlin.concurrent.thread
 
 object Dashboard {
+    private val limelightAddrs = ArrayList<String>()
+    private var limelightsConnected = true
+
     init {
+        limelightAddrs.add("10.36.36.11") // TODO: Think of a better way to do this
+
+        thread(isDaemon = true) {
+            while (true) {
+                var allConnected = true // Prevent the value from flickering
+                for (addr in limelightAddrs) {
+                    try {
+                        InetAddress.getByName(addr).isReachable(1000)
+                    } catch (err: Exception) {
+                        allConnected = false
+                        break
+                    }
+                }
+                limelightsConnected = allConnected
+                Thread.sleep(2000)
+            }
+        }
+
```

Comment:
Do we need to recheck isDamon? Can it be combined or could we just return if false?

Reply:
`isDaemon` is a named parameter to a function and isn't being checked by us - although we could combine these 2 threads into one single "diagnostics" thread